plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx

```diff
@@ -39,6 +42,9 @@ interface ExecuteWorkflowPageProps {
 export const ExecuteWorkflowPage = (props: ExecuteWorkflowPageProps) => {
   const orchestratorApi = useApi(orchestratorApiRef);
   const { workflowId } = useRouteRefParams(executeWorkflowRouteRef);
+  const { businessKey } = useRouteRefParams(
```

Comment:
Instead of creating a new route, I believe you can use the line above as follows:
```diff
- const { workflowId } = useRouteRefParams(executeWorkflowRouteRef);
+ const executeParams = useRouteRefParams(executeWorkflowRouteRef);
```
Then, `executeParams.businessKey` could have some value or be `undefined`.
You can use `executeParams.workflowId` in the other places.

Wdyt? Please let me know if this works

Reply:
Unfortunately, this approach doesn't work as expected.